jscomp/bsb/bsb_config_parse.ml

```diff
@@ -79,6 +79,11 @@ let extract_main_entries (map :json_map) =
   []  
 #end
 
+let entries_from_bsconfig () = 
+  let json = Ext_json_parse.parse_json_from_file Literals.bsconfig_json in
+  match json with
+    | Obj {map} -> extract_main_entries map
+    | _ -> assert false
 
```

Comment:
The json file is parsed several time, shall we add a lazy field to avoid parsing it several times like
```
let json_lazy = lazy (Ext_json_parse.parse_json_... )
```